scripts/azdo/pipeline.yml

```diff
@@ -20,5 +20,6 @@ jobs:
   - checkout: self
     submodules: true
   - script: |
+      git submodule update --remote --checkout
       source ni-oe-init-build-env
```

Comment:
I've pulled the PR which updates this init script with the correct (new) environment variables. Change this source line to [this file](https://github.com/ni/nilrt/blob/nilrt/master/dunfell/nilrt-build-init.env) to populate `NILRT_ROOT`.